#goal
string cart_name
---
#result
bool cart_placed
string action_status
---
#feedback